crates/bevy_internal/src/default_plugins.rs

```diff
@@ -26,16 +26,27 @@ pub struct DefaultPlugins;
 
 impl PluginGroup for DefaultPlugins {
     fn build(self) -> PluginGroupBuilder {
-        let mut group = PluginGroupBuilder::start::<Self>();
-        group = group
-            .add(bevy_log::LogPlugin::default())
-            .add(bevy_core::CorePlugin::default())
-            .add(bevy_time::TimePlugin::default())
-            .add(bevy_transform::TransformPlugin::default())
-            .add(bevy_hierarchy::HierarchyPlugin::default())
-            .add(bevy_diagnostic::DiagnosticsPlugin::default())
-            .add(bevy_input::InputPlugin::default())
-            .add(bevy_window::WindowPlugin::default());
+        let mut group = PluginGroupBuilder::start::<Self>().add(bevy_core::CorePlugin::default());
+
+        #[cfg(feature = "bevy_log")]
+        {
+            group = group.add(bevy_log::LogPlugin::default());
```

Comment:
It is better to add `LogPlugin` first, as anything logged before its initialization will not get displayed.